include/sdf/InterfaceModel.hh

```diff
@@ -124,10 +124,21 @@ class SDFORMAT_VISIBLE InterfaceModel
   /// \brief Gets registered links.
   public: const std::vector<sdf::InterfaceLink> &Links() const;
 
+  /// \brief Whether the custom parser supports merge-include.
+  /// \return If set by the custom parser, return the set value, otherwise
+  /// nullopt.
```

Comment:
Unclear what `nullopt` implies. From reading code, it seems like it implies `false`.

Why not just return `bool`? (is there any logic for checking if this is unset?)

Reply:
Currently, both `nullopt` and `false` are treated the same way, but my intention was that this could be used to distinguish between a custom parser that explicitly does not support merge-include and one that was developed before the merge-include feature was introduced. Since custom parsers are fairly new, there's probably not much value in the distinction. Changed to bool in https://github.com/ignitionrobotics/sdformat/pull/768/commits/1907e5de87c02c1d38d8347030b8d0e177a9efca

Reply:
OK Thanks!